opencontext/storage/backends/__init__.py

```
@@ -12,6 +12,6 @@
 from .chromadb_backend import ChromaDBBackend
 
 try:
-    __all__ = ['SQLiteBackend', 'ChromaDBBackend', 'VikingDBBackend']
+    __all__ = ['SQLiteBackend', 'ChromaDBBackend']
```

Ruff (0.13.3)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)